python/lsst/ts/MTAlignment/mock/mock_t2sa.py

```diff
@@ -37,6 +38,10 @@
 T2SA_STATUS_READY = "READY"
 T2SA_REPLY_READY = "READY"
 
+# Special status string for busy; may be any reasonable value
+# that does not match a non-busy status.
+BUSY_STATUS = "_BUSY_"
```

Comment:
Maybe use the value from `constants.py` instead?

Reply:
I was reluctant to do that because the two values are only indirectly connected. But it's now a moot point because constants.py is gone as a result of the change mentioned above.